### ISSUE TITLE
cp-kafka error Failed to construct kafka producer

### ISSUE THREAD

**Issue by willis0591** [confluentinc/cp-helm-charts#559]
install with k8s, k8s version 1.18, cp-kafka version 6.1.0
6.2.0 the same error!
![image](https://user-images.githubusercontent.com/37576675/132482224-eb31ed84-5b39-4dbf-a46f-1019dd75ed05.png)
![image](https://user-images.githubusercontent.com/37576675/132482326-57074dc7-09c3-43dc-91d9-a43fc9f58c98.png)
[kafka.log](https://github.com/confluentinc/cp-helm-charts/files/7127559/kafka.log)
[kafka.txt](https://github.com/confluentinc/cp-helm-charts/files/7127573/kafka.txt)

**Comment by willis0591** [comment]
yes, I change service config , has been resolved, now I add install all project, schema-registry error
![image](https://user-images.githubusercontent.com/37576675/133184990-a6a7f430-91c1-4cb7-aa0c-eb8af7f9a962.png)

**Comment by willis0591** [comment]
yes, I change service config, has been resoled, now I install other all, schema-registry error
![image](https://user-images.githubusercontent.com/37576675/133185237-32c95940-8ea0-4939-83f2-6dc56f918d43.png)
![image](https://user-images.githubusercontent.com/37576675/133185336-aa5e074d-2ee6-4dd4-ba74-6be9748ca5a8.png)

**Comment by OneCricketeer** [comment]
See #548 for a fix
